src/pages/ContactForm/ContactForm.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import Container from "../../components/Container/Container";
+import { ScrollView, TextInput,View,Text,Pressable } from "react-native";
+import GoBackButton from "../../components/GoBackButton";
+import { Formik } from "formik";
+import DropdownComponent from './DropdownComponent';
+
+function ContactForm({ navigation }) {
+   
+
+  const formInitialValues = {
+    nameAndSurname: "",
+    email: "",
+  };
+    
+    return (
+
+      <Container customClasses="px-2">
+        <GoBackButton navigation={navigation} header="İletişime Geçin" />
+             <View className="flex-[1]"></View>
+             <Formik
+        initialValues={formInitialValues}
+        onSubmit={(values) => console.log(values)}
+      >
+        {({ handleChange, handleBlur, handleSubmit, values }) => (
+          <View className="px-4 flex-[10]">
+            <View className="gap-4 flex-[10]">
+              <View className="gap-2">
```

Comment:
If you set gap like this, this gap value can create spacing both x and y axis. No need spacing at x axis so "gap-y-4" is ideal.